data/human/free worlds 4 epilogue.txt

```diff
@@ -144,7 +144,7 @@ mission "FW Epilogue: Freya"
 				`	"You sound frustrated."`
 				`	"You know, the war is over, you can relax if you want to."`
 			`	"I know, I know," she says. "I complain, but really there's nowhere else I would rather be. Even if everything we're discovering here is nothing more than what they intended for us to discover, it's still the most interesting work I could possibly be doing right now. So, don't feel sorry for me. Really."`
-			`	You talk for a while longer, and she tells you she'll be sure to be in touch if they ever get to the root of the Pug's motivation. "Until then," she says, "good luck in your travels. I hope the galaxy never runs out of ways to surprise you."`
+			`	You talk for a while longer, and she tells you she'll be sure to be in touch if she ever gets to the root of the Pug's motivation. "Until then," she says, "good luck in your travels. I hope the galaxy never runs out of ways to surprise you."`
```

Comment:
The plural is probably referring to all the people investigating the Pug technology, not, a singular reference to Freya herself. The line immediately above this has her referring to them as a plural as well.